Services/Mail/classes/Address/Parser/RFC822.php

```diff
@@ -891,10 +891,12 @@ protected function _validateLocalPart($local_part)
 
         // Validate each word.
         foreach ($words as $word) {
+            // mjansen patch 16 Sep 2015 start
```

Comment:
@iszmais I prefer something like

`iszmais patch 19 May 2020 start` and `iszmais patch 19 May 2020 end`
 
:-)

FYI: The patch markers should be used here because this class was copied from the`PEAR` library.